### PR TITLE
Add PoolShare component

### PR DESCRIPTION
**Pull request by nomadnic**

### CODE REVIEW

**Comment by nomadnic** [comment]
Closes #101

**Comment by nomadnic** [comment]
This component is a subset of the PoolStake component  in bepswap. 

I rewrote the implementation/styling to be more aligned with asgardex. This commit is essentially the difference between what was in bepswap and my implementation: https://github.com/thorchain/asgardex-electron/pull/143/commits/20bb34a44bce22be2b4082e4f4c320f30474e61c

@veado 
I'm not 100% sure on the types used for the different values (..Share/..Price) in PoolShare.tsx. Maybe you can comment here.

File: src/components/uielements/label/label.style.ts [hunk 1]
```diff
@@ -48,6 +49,7 @@ const colors: Colors = {
 
 export const LabelWrapper = styled.div`
   padding: 10px 0;
+  text-align: ${(props: Props) => props.align};
```
Comment (FrozenIce0617) [inline src/components/uielements/label/label.style.ts:52]: `props.align` can be `undefined`.
I think it would be great if you can add default value here.

Comment (nomadnic) [reply src/components/uielements/label/label.style.ts:52]: The default is defined in the src/components/uielements/label/label.tsx component. This should be good enough right? See https://github.com/thorchain/asgardex-electron/blob/20bb34a44bce22be2b4082e4f4c320f30474e61c/src/components/uielements/label/label.tsx#L20

Comment (veado) [reply src/components/uielements/label/label.style.ts:52]: @FrozenIce0617 is right (good catch btw). A default value in `label.tsx` won't save you, as long as `align?: TextAlignment` is defined as an optional value.

Comment (FrozenIce0617) [reply src/components/uielements/label/label.style.ts:52]: If the `align` has a default value, we can define `align` as a required props. :)

File: src/components/uielements/label/label.style.ts [hunk 2]
```diff
@@ -55,4 +56,5 @@ export const LabelWrapper = styled.div<Props>`
   letter-spacing: ${({ size = 'normal' }) => fontSettings[size].spacing};
   color: ${({ color }) => colors[color || 'normal']};
   cursor: ${({ onClick }) => onClick && 'pointer'};
+  text-align: ${(props: Props) => props.align || 'left'};
```
Comment (veado) [inline src/components/uielements/label/label.style.ts:59]: [hint] 
```ts
text-align: ${({ align = 'left' }) => align};
```

**Pull request merged**